docs/sources/setup-grafana/configure-security/configure-authentication/github/index.md

```diff
@@ -159,6 +161,25 @@ role_attribute_path = "'Viewer'"
 skip_org_role_sync = false
 ```
 
+#### Org roles mapping examples
+
+##### Map organization roles
+
+Given a user member of the `@my-github-organization/my-github-team1` and `@my-github-organization/my-github-team2` Gitub teams.
```

Comment:
```suggestion
Let's use the following Github teams: `@my-github-organization/my-github-team1` and `@my-github-organization/my-github-team2`.
```

Reply:
You can also say "Let's say we have the following Github teams: `@my-github-organization/my-github-team1` and `@my-github-organization/my-github-team2`.